var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -78,6 +80,15 @@ def cmake_args(self):
             args.append('-Denable_openmp=OFF')
             args.append('-DCMAKE_DISABLE_FIND_PACKAGE_OpenMP=ON')
 
+        if '+cuda' in spec:
+            args.append('-DTPL_ENABLE_CUDALIB=TRUE')
+            args.append('-DTPL_CUDA_LIBRARIES=-L%s -lcublas -lcudart'
+                        % spec['cuda'].prefix.lib)
```

Comment:
```suggestion
            args.append('-DTPL_CUDA_LIBRARIES=' + 
                        spec['cuda'].libs.ld_flags)
```
This will work even if the libs are installed in `lib64` instead of `lib`.

Reply:
This patch does not work. I get:
```
'-DTPL_CUDA_LIBRARIES=-L/data/balay/spack-xsdk-slu2/opt/spack/linux-centos7-skylake_avx512/gcc-7.4.0/cuda-10.2.89-xdkte4bqy4qbf5wqr2r3wxgen6uuiiuv/lib64 -lcudart'
```
i.e `-lcublas` is missing.

Reply:
Should we add `libcublas` to the `libs` method of the CUDA package? @ax3l @Rombur

Reply:
Ouch, probably not a bad idea.

Reply:
I wonder why this is needed, this is a CMake package and CMake is quite good in picking these libs up.

Reply:
SuperLU_dist's CMakeLists.txt does not look for cuBLAS, it expects the user to set `-DTPL_CUDA_LIBRARIES`